WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/CardReaderSettingsConnectedViewController.swift

```diff
@@ -70,6 +70,9 @@ private extension CardReaderSettingsConnectedViewController {
         guard let viewModel = viewModel else {
             return
         }
+
+
+
```

Comment:
Was all this whitespace intentional?

```suggestion

```

Reply:
IIRC, SDK 1.4 only updates the battery level on connection. We've asked Stripe to improve this. Not sure if that is still the case with SDK 2 but I will check.

Reply:
I'll fix the whitespace. Not intentional

Reply:
Fixed whitespace in https://github.com/woocommerce/woocommerce-ios/pull/5133/commits/82bdc6fb2399c49f055a23934b0f1d945d83be95